modules/odc-visualizing-the-binding-connections-to-resources.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc
+:_content-type: PROCEDURE
+[id="odc-visualizing-the-binding-connections-to-resources_{context}"]
+= Visualizing the binding connections to resources
+
+As a user, you can visualize a Service Binding through *Label Selector* in *Topology* view. Label selector is a feature added to Service Binding Operator that aims to simplify the process of binding applications to backing services. It does this by allowing developers to use specified labels to find and connect applications instead of using the name of the application. It aims to alleviate the confusing nature of one or many services to many application binds. This is done by specifying labels that will be consumed by the Service Binding Operator when creating `ServiceBinding` resources.
+[TIP]
+====
+When you click on the label selector associated with the Service Binding, you can navigate to a list of all connected resources.
+====
+
+To view the *Label Selector*, consider the following approaches:
+
+** After you import yaml page, you can view the *Label Selector* associated with the Service Binding in the *ServiceBinding details* page.
+
++
+.Service Binding details page
+image::odc-label-selector-sb-details.png[]
+
+[NOTE] 
+====
+To use *Label Selector* and to create one or more connections at once, you must import `ServiceBinding` resource yaml.
+====
+
+** After you click on the dangling arrow, the Service Binding connector *Details* side-panel appears. You can view the *Label Selector* associated with it here.
```

Comment:
Will this be a dangling arrow as the connection is established? As seen in the [doc](https://docs.openshift.com/container-platform/4.12/applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.html#odc-creating-a-visual-connection-between-components_odc-connecting-an-application-to-a-service-using-the-developer-perspective) dangling is used for the arrow appearing when the user hovers the mouse pointer on the application node i.e. before the connection is established

Reply:
@sanketpathak I think this a drag and drop arrow. I'll make the correction.